sfr/subchannel/multiple_SCM_assemblies/7assemblies/abr_7assemblies.i

```diff
@@ -0,0 +1,321 @@
+    duct_block_ids = '1003  1006'
+    outward_interface_boundary_names = 'wall_in_02 wall_out_02'
+    interface_boundary_id_shift = 300
+  []
+
+  [XX03]
+    type = PolygonConcentricCircleMeshGenerator
+    num_sides = 6
+    num_sectors_per_side = '${ns} ${ns} ${ns} ${ns} ${ns} ${ns}'
+    background_intervals = 1
+    background_block_ids = '15'
+    polygon_size = '${fparse outer_duct_out / 2 + inter_wrapper_width / 2}'
+    duct_sizes = '${fparse outer_duct_in / f /2} ${fparse outer_duct_out / f / 2}'
+    duct_intervals = ${duct_intervals_perishperic}
+    duct_block_ids = '1003  1006'
+    outward_interface_boundary_names = 'wall_in_03 wall_out_03'
+    interface_boundary_id_shift = 400
+  []
+
+  [XX04]
+    type = PolygonConcentricCircleMeshGenerator
+    num_sides = 6
+    num_sectors_per_side = '${ns} ${ns} ${ns} ${ns} ${ns} ${ns}'
+    background_intervals = 1
+    background_block_ids = '16'
+    polygon_size = '${fparse outer_duct_out / 2 + inter_wrapper_width / 2}'
+    duct_sizes = '${fparse outer_duct_in / f /2} ${fparse outer_duct_out / f / 2}'
+    duct_intervals = ${duct_intervals_perishperic}
+    duct_block_ids = '1003  1006'
+    outward_interface_boundary_names = 'wall_in_04 wall_out_04'
+    interface_boundary_id_shift = 500
+  []
+
+  [XX05]
+    type = PolygonConcentricCircleMeshGenerator
+    num_sides = 6
+    num_sectors_per_side = '${ns} ${ns} ${ns} ${ns} ${ns} ${ns}'
+    background_intervals = 1
+    background_block_ids = '17'
+    polygon_size = '${fparse outer_duct_out / 2 + inter_wrapper_width / 2}'
+    duct_sizes = '${fparse outer_duct_in / f /2} ${fparse outer_duct_out / f / 2}'
+    duct_intervals = ${duct_intervals_perishperic}
+    duct_block_ids = '1003  1006'
+    outward_interface_boundary_names = 'wall_in_05 wall_out_05'
+    interface_boundary_id_shift = 600
+  []
+
+  [XX06]
+    type = PolygonConcentricCircleMeshGenerator
+    num_sides = 6
+    num_sectors_per_side = '${ns} ${ns} ${ns} ${ns} ${ns} ${ns}'
+    background_intervals = 1
+    background_block_ids = '18'
+    polygon_size = '${fparse outer_duct_out / 2 + inter_wrapper_width / 2}'
+    duct_sizes = '${fparse outer_duct_in / f /2} ${fparse outer_duct_out / f / 2}'
+    duct_intervals = ${duct_intervals_perishperic}
+    duct_block_ids = '1003  1006'
+    outward_interface_boundary_names = 'wall_in_06 wall_out_06'
+    interface_boundary_id_shift = 700
+  []
+
+  [pattern]
+    type = PatternedHexMeshGenerator
+    inputs = 'XX00 XX01 XX02 XX03 XX04 XX05 XX06'
+    pattern = '5 4 ;
+              6 0 3 ;
+               1 2'
+    pattern_boundary = none
+  []
+
+  [extrude]
+    type = AdvancedExtruderGenerator
+    direction = '0 0 1'
+    input = pattern
+    heights = '${height}'
+    num_layers = '${n_ax}'
+  []
+
+  [inlet_interwrapper]
+    type = ParsedGenerateSideset
+    input = extrude # inlet_interwall
+    combinatorial_geometry = 'abs(z) < 1e-6'
+    included_subdomains = '1006 1003'
```

Comment:
why the walls too?
also please use names in the script. this is not maintainable with these IDs everywhere